tests/rptest/util.py

```diff
@@ -19,8 +19,9 @@ class Scale:
     LOCAL = "local"
     CI = "ci"
     RELEASE = "release"
+    CDT = "cdt"
```

Comment:
I'm not sure we should call clustered ducktape a "scale": we might still want to distinguish between short and longer runs of a test (like cs vs release) in the context of the higher-powered runtime environment.

In #4376 I'm using release scale as the indicator to run things for a long time, and adding an explicit "dedicated_nodes" boolean that tells tests whether they're in an environment that has dedicated cores and memory for each redpanda node -- I think this are probably two axes rather than one thing to conflate.

Reply:
Agreed.. scale in "scale test" to me means runtime; your set of tests that are slow. Totally not intuitive--the context:
You may have fast & wide tests (e.g. a 5 minute distributed test suite that can light up 30 full servers).. that is a regular CI / premerge test. Scale tests are usually the suite that takes forever to run (and might use really large clusters too) which you don't do for CI--maybe as a weekly or per-release

Reply:
I agree the change in 4376 makes more sense.